### PR TITLE
Add support for creating fohm names for mutant fastq in cg

### PR DESCRIPTION
**Pull request by Mropat**
## Description

Links fastq files to mutant working dir with FOHM standard names

### Expected test outcome
- [x] check that files are named correctly

## Review
- [x] code approved by PG
- [x] tests executed by MR
- [x] "Merge and deploy" approved by MR
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan
- [ ] Deploy this branch

### CODE REVIEW

File: cg/meta/workflow/fastq.py [hunk 1]
```diff
@@ -268,10 +271,35 @@ def create_fastq_name(
...
+    ) -> str:
+        """Name a FASTQ file following mutant conventions. Naming must be
+        xxx_R_1.fastq.gz and xxx_R_2.fastq.gz"""
+        # ACC1234A1_FCAB1ABC2_L1_1.fastq.gz sample_flowcell_lane_read.fastq.gz
+
+        return f"{flowcell}_L{lane}_{meta}_{read}.fastq.gz"
+
+    @staticmethod
+    def get_concatenated_name(linked_fastq_name: str) -> str:
+        """"create a name for the concatenated file for some read files"""
```
Comment (patrikgrenfeldt) [inline cg/meta/workflow/fastq.py:304]: "for some read files" ?
sounds a bit non-deterministic :)

**Review by patrikgrenfeldt — APPROVED** [review]
👍

File: cg/meta/workflow/fastq.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/meta/workflow/fastq.py]: *the* read files?

Comment (patrikgrenfeldt) [reply cg/meta/workflow/fastq.py]: Ok, perfect

**Review by sylvinite — APPROVED** [review]

**Comment by Mropat** [comment]
Deployed on prod

```
Building wheels for collected packages: cg
  Building wheel for cg (PEP 517) ... done
  Created wheel for cg: filename=cg-20.15.1-py3-none-any.whl size=405539 sha256=9c9c887a05f4d14a612a2bd9027197fb24736df5c176bcc5a34078564e432dfa
  Stored in directory: /home/maryia.ropart/tmp/pip-ephem-wheel-cache-98zhlrdk/wheels/8c/1e/b8/816ed4facf8c5628b55966d3f2448eeb6ca70f35deff59389e
Successfully built cg
Installing collected packages: cg
  Attempting uninstall: cg
    Found existing installation: cg 20.15.0
    Uninstalling cg-20.15.0:
      Successfully uninstalled cg-20.15.0
Successfully installed cg-20.15.1

```

**Pull request merged**